README.md

```diff
@@ -13,15 +13,23 @@ Windows software for hosting locally connected USB devices to other machines, in
 This software requires Microsoft Windows 8 / Microsoft Windows Server 2012 or newer;
 it does not depend on any other software.
 
-1) Run the installer (.msi) from the <a href="https://github.com/dorssel/usbipd-win/releases/latest">latest release</a>
-   on the Windows machine where your USB device is connected.
+### Installation
 
-   Alternatively, use the Windows Package Manager:
-   <pre>
-   winget install usbipd
-   </pre>
+Run the installer (.msi) from the <a href="https://github.com/dorssel/usbipd-win/releases/latest">latest release</a> on the Windows machine where your USB device is connected. 
+
+Alternatively, use the Windows Package Manager:
+```
+winget install usbipd
+```
+This will install and run a service called `usbpd-win`. You can check the status of this service using the Services app from Windows. Additionally, it will add `UsbIpServer` in your path.
+
+### Lookup and Enable Devices
+
+By default devices are not shared with usbip clients. To lookup and share devices, open a command prompt and use `UsbIpServer` tool.
```

Comment:
The new default (i.e. not sharing) is a big improvement (at least security wise). But now users will have to use the UsbIpServer.exe executable from the command prompt to get anything to work. We might as well tell them where to find it:

`%ProgramFiles%\usbip-win\UsbIpServer.exe`

Reply:
Would adding the tool to PATH be more user friendly?

Then they would only need to type usbipserver in a command prompt.

Reply:
Yes, that could be done. I'll add that to WiX as well. Now *could* be a good time to rename it to `usbipd.exe`, what do you think?

Reply:
Great! I think so too